fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/utils/SubjectMappingUtil.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Licensed to DuraSpace under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * DuraSpace licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.fcrepo.kernel.api.utils;
+
+import static org.apache.jena.graph.NodeFactory.createURI;
+
+import org.apache.jena.graph.Node;
+import org.apache.jena.graph.Triple;
+
+/**
+ * Utility for remapping subjects in rdf triples.
+ *
+ * @author bbpennel
+ */
+public class SubjectMappingUtil {
+
+    private SubjectMappingUtil() {
+        // Empty constructor for static utility class
+    }
+
+    /**
+     * @param t
+     * @param resourceUri
+     * @param destinationUri
+     * @return
+     */
+    public static Triple mapSubject(final Triple t, final String resourceUri, final String destinationUri) {
+        final Node destinationNode = createURI(destinationUri);
+        return mapSubject(t, resourceUri, destinationNode);
+    }
+
+    /**
+     * @param t
```

Comment:
[WARNING] /home/awoods/programming/java/apps/fedora/futures/fcrepo4/fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/utils/SubjectMappingUtil.java:48: warning: no description for @param
[WARNING] * @param t